neofetch

```diff
@@ -1634,10 +1640,10 @@ get_term() {
     esac
 
     # Check $PPID for terminal emulator.
-    while [[ -z "$term" ]]; do
-        parent="$(get_ppid "$parent")"
-        name="$(get_process_name "$parent")"
+    while [[ -z "$term" ]] || ((parent > 1)); do
 
```

Comment:
[This one](https://github.com/Takeya-Yuki/neofetch/blob/4b0cf233a86e150766cf6f45d18045a3e71a29d7/neofetch#L1644).